input/resources/StructureDefinition-name-context.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="name-context"/>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/>
+  </extension>
+  <url value="http://hl7.org.au/fhir/StructureDefinition/name-context"/>
+  <name value="NameContext"/>
+  <title value="Name Context"/>
+  <status value="active"/>
+  <description value="A suitable context of use for a patient, person, practitioner or related person name can be indicated using this extension. &#xA;This includes an ability to provide a coded concept or text description of a suitable use of the associated name.&#xA;The usage described may be varied in nature including suitability for a particular purpose e.g. IHI validation, or highlighting (in)appropriate&#xA;use when referring to an individual e.g. deceased Aboriginal name, transgender deadname."/>
```

Comment:
Should be in format: "This extension applies to the <resource/element/datatype> and is used to indicate/represent .."

Reply:
reworded as HumanName datatype extension and mentioning the resource type content; similar to medication-type